File: pset1/mario/mario.c
#include <stdio.h>
#include <cs50.h>

int get_positive_int(void);

int main(void)
{
    //draw hashes
    int height = get_positive_int();
    for (int i = 1; i <= height; i++)
    {
        for (int j = 0; j < height - i; j++)
        {
            printf(" ");
        }
        for (int k = 0; k < i; k++)
        {
            printf("#");
        }
        printf("  ");
        for (int k = 0; k < i; k++)
        {
            printf("#");
        }
        printf("\n");
    }

}

//get integer from 1 to 8
int get_positive_int(void)
{
    int n;
    do
    {
        n = get_int("Height: ");
    }
    while (n < 1 || n > 8);
    return n;
}

File: pset6/mario/more/mario.py
from cs50 import get_int


def main():
    i = get_positive_int()
    for a in range(1, i + 1):
        print(" " * (i - a) + "#" * a + "  " + "#" * a)
        a += 1


def get_positive_int():
    h = 0
    while True:
        h = get_int("Height: ")
        if (1 <= h and h <= 8):
            break
    return h


main()